app/src/lang/translations/en-US.yaml

```
@@ -187,6 +187,7 @@ permissions_for_role: 'Items the {role} Role can {action}.'
 fields_for_role: 'Fields the {role} Role can {action}.'
 validation_for_role: 'Field {action} rules the {role} Role must obey.'
 presets_for_role: 'Field value defaults for the {role} Role.'
+presets_field_warning: 'The app interface for relational field {field} only supports the detailed syntax.'
```

```suggestion
presets_field_warning: 'Relational presets for field "{field}" should be configured with the "detailed" syntax.'
```

Should make it a tad more obvious that "detailed" is a word we call it so it's easier to search for it in the docs or ask a question about it